File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/selenium/input-events")

    actions = webdriver.ActionChains(driver)

    print(driver.title)

    actions.send_keys("S")

    actions.key_down(Keys.CONTROL)

    actions.send_keys("a")

    actions.send_keys("c")

    actions.key_up(Keys.CONTROL)

    actions.perform

    driver.close()

File: Selenium/Activities/Activity6.java
package activities;

import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		Actions builder = new Actions(driver);
		
		driver.get("https://v1.training-support.net/selenium/input-events");
		
		System.out.println(driver.getTitle());
		
		builder.sendKeys("S").build().perform();
		
		builder
		.keyDown(Keys.CONTROL)
		.sendKeys("a")
		.sendKeys("c")
		.keyUp(Keys.CONTROL)
		.build()
		.perform();
		
		driver.close();

	}

}
